cmd/proxy/actions/app.go

```diff
@@ -130,6 +130,24 @@ func App() (*buffalo.App, error) {
 }
 
 func getWorker(ctx context.Context, s storage.Backend, mf *module.Filter) (worker.Worker, error) {
+	if env.RedisMockInMem() {
+		return registerInMem(ctx, s, mf)
+	}
+	return registerRedis(ctx, s, mf)
+}
+
+func registerInMem(ctx context.Context, s storage.Backend, mf *module.Filter) (worker.Worker, error) {
+	w := worker.NewSimple()
+	if err := w.Register(FetcherWorkerName, GetProcessCacheMissJob(ctx, s, w, mf)); err != nil {
+		return nil, err
+	}
+	if err := w.Register(ReporterWorkerName, GetCacheMissReporterJob(w, mf)); err != nil {
```

Comment:
no need for this one

Reply:
basically proxy is sync so it does not need workers now.
please keep one in Olympus
@marwan-at-work correct me if i'm wrong

Reply:
I think we can still make use of the worker in proxy. If the proxy doesn't have the requested module we could redirect go to Olympus and at the same time queue a job to fill the cache. Otherwise go would have to wait for Olympus to download the module, store it in cache, then for proxy to download it, store it in it's own cache and then send to go.

Reply:
@kdvy @michalpristas @marpio @marwan-at-work are you all ok with removing this for now? It's essentially dead code until we implement the redirect logic that @marpio mentioned in ☝️

Reply:
i'm ok with removing, more dead code == less readable code